client/src/components/HostSection.jsx

```diff
@@ -1,8 +1,14 @@
-import React from 'react';
+import React, { useState } from 'react';
```

Comment:
Thanks for making me learn about React Hooks :P

Reply:
Oh yeah, haha. It's for the best 😁